### PR TITLE
Return 400 BadRequest when X-Broker-API-Version in a request is missing

### PR DESCRIPTION
**Pull request by aukhatov**
Hi guys!
The PR is related with #217 and https://github.com/openservicebrokerapi/servicebroker/issues/673

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@aukhatov Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-open-service-broker&pullRequestId=219)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-open-service-broker&pullRequestId=219) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@aukhatov Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-open-service-broker&pullRequestId=219)!

File: spring-cloud-open-service-broker-core/src/main/java/org/springframework/cloud/servicebroker/controller/ServiceBrokerExceptionHandler.java [hunk 1]
```diff
@@ -20,21 +20,7 @@
...
-import org.springframework.cloud.servicebroker.exception.ServiceBrokerInvalidParametersException;
-import org.springframework.cloud.servicebroker.exception.ServiceBrokerOperationInProgressException;
-import org.springframework.cloud.servicebroker.exception.ServiceBrokerUnavailableException;
-import org.springframework.cloud.servicebroker.exception.ServiceDefinitionDoesNotExistException;
-import org.springframework.cloud.servicebroker.exception.ServiceInstanceBindingDoesNotExistException;
-import org.springframework.cloud.servicebroker.exception.ServiceInstanceBindingExistsException;
-import org.springframework.cloud.servicebroker.exception.ServiceInstanceDoesNotExistException;
-import org.springframework.cloud.servicebroker.exception.ServiceInstanceExistsException;
-import org.springframework.cloud.servicebroker.exception.ServiceInstanceUpdateNotSupportedException;
+import org.springframework.cloud.servicebroker.exception.*;
```
Comment (scottfrederick) [inline spring-cloud-open-service-broker-core/src/main/java/org/springframework/cloud/servicebroker/controller/ServiceBrokerExceptionHandler.java]: Please don't use wildcard imports.

Comment (aukhatov) [reply spring-cloud-open-service-broker-core/src/main/java/org/springframework/cloud/servicebroker/controller/ServiceBrokerExceptionHandler.java]: Thank you. I fixed :)

**Comment by royclarkson** [comment]
Closed in dcce56b32affb70c7efaeeaa3d8a127557dd8ea9 and b6a571f9b82fdcdd0fb781f4ba34ccd862683d69.

Resolves #217

**Pull request closed without merging**